### ISSUE TITLE
currency() when there's no fetchCurrencies

### ISSUE THREAD

**Issue by Deathamns** [ccxt/ccxt#1161]
`exchange.currency(code)['id']` returns incorrect id for exchanges that don't have `fetchCurrencies` implementation and have ambiguous currency names.

```js
let binance = new ccxt['binance']();
await binance.loadMarkets();
let currency = binance.currencies['BCH'];
// second is correct
console.log(currency.code, binance.commonCurrencyCode('BCC'));
console.log(currency.id, binance.currencyId('BCH'));
```

**Comment by kroitor** [comment]
> returns incorrect id

Are you sure you're not confusing the purpose of those functions? 

- `currencyId()` is for fetching the currency *id* by currency *code* (your second line)
- `commonCurrencyCode()` is for converting exchange-specific currency *code* to a common currency *code* (it does not return ids at all).

Does this answer your question? )

**Comment by Deathamns** [comment]
Probably I wasn't clear enough.

```js
console.log(exchange.currency('BCH'));
// { id: 'BCH', code: 'BCH', precision: 8 }
```
The id there should be `BCC`.

**Comment by kroitor** [comment]
As always, thanks for your help, fixed in 1.10.629.
